server/views/components/actions-card/actionCards.test.njk

```diff
@@ -3,5 +3,6 @@
 {{ actionsCard({
   prisonerSummary: prisonerSummary,
   userHasPermissionTo: userHasPermissionTo,
-  educationSupportPlanCreationStatus: educationSupportPlanCreationStatus
+  educationSupportPlanCreationStatus: educationSupportPlanCreationStatus,
+  actionMenuType: actionMenuType
```

Comment:
Added the actionMenuType to the main test file (rather than having one for each type), as this is how we expect the real nunjucks templates to call this component at runtime